packages/isomorphic/src/utils/index.ts

```diff
@@ -0,0 +1,28 @@
+import { randomBytes as cryptoRandomBytes } from 'crypto'
+import { BytesToHexFn, HexToBytesFn, RandomBytesFn } from './types'
+
+// Typed to ensure uniformity between node and browser implementations and docs
+/* eslint-disable func-style, eslint-comments/disable-enable-pair -- see above */
+const bytesToHex: typeof BytesToHexFn = (bytes: Uint8Array | number[]) => {
+  return Buffer.from(
+    bytes instanceof Uint8Array ? bytes : Uint8Array.from(bytes),
+  )
+    .toString('hex')
+    .toUpperCase()
+}
+
+const hexToBytes: typeof HexToBytesFn = (hex: string): Uint8Array => {
+  return toUintArray(Buffer.from(hex, 'hex'))
+}
+
+const randomBytes: typeof RandomBytesFn = (size: number): Uint8Array => {
+  return new Uint8Array(cryptoRandomBytes(size).buffer)
+}
+
+function toUintArray(buf: Buffer): Uint8Array {
```

Comment:
I saw a couple spots where `UInt` was lowecase to `Uint` - I'd recomend doing a global search with capitalization on to verify that all instances capitalize the `Int` part. (Also added the 8 since I think the size matters for this)
```suggestion
function toUInt8Array(buf: Buffer): Uint8Array {
```

Reply:
This one is complicated.  `Uint8Array` is the type name that is built into JS.  We have previously existing classes in binary-codec that uses `UInt*` naming.  Additionally Buffer which I am hoping to get rid of uses capital `I` as well.  Long story short built in types (Buffer is native to node) arent consistent either.  When referencing a type like this particular case it should mirror what the type is so lowercase.  For the classes in binary codec It can be left as is because it is referencing the concept of a UInt.

I will rename this one to include the 8.